docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -44,6 +44,7 @@ You can add these Kubernetes annotations to specific Ingress objects to customiz
 |[nginx.ingress.kubernetes.io/limit-rps](#rate-limiting)|number|
 |[nginx.ingress.kubernetes.io/permanent-redirect](#permanent-redirect)|string|
 |[nginx.ingress.kubernetes.io/permanent-redirect-code](#permanent-redirect-code)|number|
+|[nginx.ingress.kubernetes.io/temporal-redirect](#temporal-redirect)|string|
```

Comment:
"-code" is missing in the anchor.

Reply:
OK, this is not a code so I'll correct the right place